### ISSUE TITLE
Add support for module routing (also feature modules)

### ISSUE THREAD

**Issue by mackelito** [BeastCode/VSCode-Angular-TypeScript-Snippets#15]
When creating a new "modulename-routing-module.ts" file it would be great to have 2 templates.
1. RouterModule.forRoot
````
export const routes: Routes = [
  { path: '', redirectTo: 'user', pathMatch: 'full' },
  { path: 'my-account', redirectTo: 'my-account', pathMatch: 'full' }
];

@NgModule({
  imports: [RouterModule.forRoot(routes)],
  exports: [RouterModule]
})
export class AppRoutingModule { }

````

2. RouterModule.forChild (feature modules)
````
@NgModule({
  imports: [RouterModule.forChild([
    {
      path: 'my-account', component: UserComponent, children: [
        { path: '', component: DashBoardComponent },
      ]
    },
  ])],
  exports: [RouterModule]
})
````

**Comment by BeastCode** [comment]
Will add when I'm back from ng-conf.

Tack /Micke

**Comment by BeastCode** [comment]
Added snippet and published 4.0.6 to the market

**Comment by mackelito** [comment]
... ng-conf... you bastard! ;)

**Comment by BeastCode** [comment]
Hahaha. Sorry, didn't mean to make you jealous, just wanted let you know that I'm not ignoring your suggestion.

**Comment by BeastCode** [comment]
Fixed in 4.0.8:
  ng-router-appmodule
  ng-router-featuremodule
